src/source/Ice/IceAgent.c

```diff
@@ -599,7 +599,7 @@ STATUS iceAgentStartGathering(PIceAgent pIceAgent)
     CHK(!ATOMIC_LOAD_BOOL(&pIceAgent->agentStartGathering), retStatus);
 
     ATOMIC_STORE_BOOL(&pIceAgent->agentStartGathering, TRUE);
-    pIceAgent->candidateGatheringStartTime = GETTIME();
+    pIceAgent->candidateGatheringProcessStartTime = GETTIME() / HUNDREDS_OF_NANOS_IN_A_MILLISECOND;
```

Comment:
Let us not do the GETTIME() conversion to milliseconds here. We should use GETTIME() and only convert wherever the calculation is done. Comment applicable wherever this is being done in this PR.